File: 200_LeetCode_Questions/Java/D02P2_LeetCode_1295.java
//Runtime: 1 ms, faster than 97.15% of Java online submissions for Find Numbers with Even Number of Digits.
//Memory Usage: 39.1 MB, less than 74.25% of Java online submissions for Find Numbers with Even Number of Digits.

class Solution {
    public int findNumbers(int[] nums) {
        int count = 0;
        for(int num : nums){
            if(Integer.toString(num).length() % 2 == 0){
                count++;
            }
        }
        return count;
    }
}

File: 200_LeetCode_Questions/C++/D02P2_LeetCode_1295.cpp
class Solution {
public:
    int findNumbers(vector<int>& num) {
        int t;
        int count = 0;
        int count1 =0;
        for ( int i = 0; i < num.size(); i++)
        {
            while(num[i])
            {
                t = num[i] %10;
                count++;
                num[i]/=10;
            }
            if ( count % 2 == 0)
            {
                count1 +=1;
            }
            
            count = 0;
        }
        return count1;
        
    }
};